client-js/app/elements/labrad-registry.ts

```diff
@@ -220,27 +218,24 @@ export class LabradRegistry extends polymer.Base {
     event.preventDefault();
     var data = JSON.parse(event.dataTransfer.getData('text'));
     console.log('dropped', event.dataTransfer);
-    if (event.ctrlKey) {
-      if (JSON.stringify(this.path) != JSON.stringify(data.path)) {
-        //Comparing strings is lame, I know
-        console.log('copying into window', this.path, data.path);
-        var oldFullPath: string[] = data.path,
-            dialog = this.$.dragCopyDialog,
-            copyOriginName = this.$.originName,
-            copyOriginPath = this.$.originPath,
-            copyNameElem = this.$.dragCopyNameInput,
-            copyPathElem = this.$.dragCopyPathInput;
-        copyNameElem.value = data.name;
-        copyOriginName.textContent = data.name;
-        copyOriginPath.textContent = JSON.stringify(data.path);
-        copyPathElem.value = this.pathToString(this.path);
-        this.$.dragCopyDialog.open();
-        window.setTimeout(() => copyNameElem.$.input.focus(), 0);
+    if (JSON.stringify(this.path) != JSON.stringify(data.path)) {  
```

Comment:
Right but we don't want users to try and make a copy of the directory with the same name, since they can't, to it has to at least default to a unique new folder name, or gracefully handle the error of copying without changing the name. 

I guess the latter is desirable, but introduces extra logic to the front-end. 

I also guess I've never checked what happens if I try to copy a folder with the same name. Ill see how that fails.